js/tic_tac_toe.js

```diff
@@ -1,2 +1,80 @@
-// Your code here
-// Use the jQuery
+$(document).ready(function() {
+  $('.box_cell').on('click', TicTacToe.takeTurn);
```

Comment:
This is great, but can you think of a way to get that 100% inside your TicTacToe object?